scout/server/blueprints/cases/views.py

```diff
@@ -327,8 +327,8 @@ def matchmaker_match(institute_id, case_name, target):
     )
     ok_responses = 0
     for match_results in match_results:
-        match_results["status_code"] == 200
-        ok_responses += 1
+        if match_results["status_code"] == 200:
```

Comment:
OHH This was not just styling

Reply:
No, but there were not so many of those. There may be more hiding among the "complex functions", e.g. controllers etc but that is for another day. Also a few security issues, that are more of integration things, like the chanjo cert one. Wasn't that like a workaround for when cert files were unavailable on clinical-db?